test/e2e/flask/multichain-api/non-evm/wallet_invokeMethod.spec.ts

```diff
@@ -0,0 +1,161 @@
+import { strict as assert } from 'assert';
+import { isObject } from 'lodash';
+import { largeDelayMs, WINDOW_TITLES } from '../../../helpers';
+import TestDappMultichain from '../../../page-objects/pages/test-dapp-multichain';
+import {
+  DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
+  replaceColon,
+} from '../testHelpers';
+import { withSolanaAccountSnap } from '../../../tests/solana/common-solana';
+
+describe('Multichain API - Non EVM', function () {
+  const SOLANA_SCOPE = 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp';
+  describe('Calling `wallet_invokeMethod`', function () {
+    describe('signIn method', function () {
+      it('Should match selected method to the expected confirmation UI', async function () {
+        await withSolanaAccountSnap(
+          {
+            ...DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
+            title: this.test?.fullTitle(),
+          },
+          async (driver, _, extensionId) => {
+            const testDapp = new TestDappMultichain(driver);
+            await testDapp.openTestDappPage();
+            await testDapp.connectExternallyConnectable(extensionId);
+            await testDapp.initCreateSessionScopes([SOLANA_SCOPE]);
+            await driver.clickElementAndWaitForWindowToClose({
+              text: 'Connect',
+              tag: 'button',
+            });
+
+            await driver.switchToWindowWithTitle(
+              WINDOW_TITLES.MultichainTestDApp,
+            );
+
+            const invokeMethod = 'signIn';
+            await driver.clickElementSafe(
```

Comment:
there's a page object that helps with this now

test/e2e/page-objects/pages/test-dapp-multichain.ts invokeMethod()